python_modules/libraries/dagster-aws/setup.py

```diff
@@ -37,6 +37,7 @@ def get_version() -> str:
     python_requires=">=3.8,<3.13",
     install_requires=[
         "boto3",
+        "boto3-stubs[essential, ecs]",
```

Comment:
who owns these libraries? are they actively maintained?

Reply:
We have been burned many times adding dependencies which is why I am asking.

Reply:
It's very active, a few releases per month. Stubs generation is automated and thus always up to date. 

https://pypi.org/project/boto3-stubs/

https://github.com/youtype/mypy_boto3_builder